metadata/src/test/java/org/apache/kafka/metadata/migration/KRaftMigrationDriverTest.java

```diff
@@ -344,8 +365,8 @@ public void testShouldNotMoveToNextStateIfControllerNodesAreNotReadyToMigrate()
             },
             new MockFaultHandler("test"),
             quorumFeatures,
-            mockTime
-        )) {
+            metrics,
+            mockTime)) {
```

Comment:
nit: let's keep the style like:

```
foo(

) {

}
```

so in this case, the `)) {` should stay on its own line